### DISCUSSION TITLE
How can I align objects on PhotoDemon?

### DISCUSSION THREAD

**Discussion by haihao888** [tannerhelland/PhotoDemon#532] [Q&A]
hello, how can I align texts or pictures like a magnet? for example, when I try to move an object to the centre, the object tends to automatically align itself to the centre of the picture. Finally I want to say that it is a wonderful programme!

**Comment by tannerhelland** [comment]
Hello @haihao888 .  Thank you for the kind words.  I apologize that PhotoDemon does not provide a "snap like a magnet" option yet.  I hope to add this feature soon.

I usually refer to Photoshop and GIMP when adding new features.  Can you recommend any other software where you really like the snap selection behavior?  Additional references are helpful for me to understand the best way to approach a new feature.  Thank you in advance!

**Reply by haihao888** [reply]
In the meantime, thank you for your reply, about programmes I don't have any, more like using a site: Photopea, which I no longer use after I discovered PhotoDemon, but the 'magnetic function' works very well indeed and that's why I was looking on PhotoDemon and couldn't find it.

**Reply by tannerhelland** [reply]
Just wanted to pop in and mention that "magnetic snapping" is now available in PhotoDemon nightly builds.  Thank you for your patience!

**Reply by haihao888** [reply]
thank you very much man, I will try it now!
